dialtone.code-workspace

```diff
@@ -1,73 +1,87 @@
 {
-	"folders": [
-		{
-			"path": "."
-		}
-	],
-	"settings": {
-		// FILES
+  "folders": [
+    {
+      "path": "."
+    }
+  ],
+  "settings": {
+    // FILES
     "files.exclude": {
       "**/.git": true,
       "**/.DS_Store": true
     },
 
-		// SEARCH
+    // SEARCH
     // Note that gitignored files are automatically excluded from search. Do not add files
     // here that are already gitignored.
     "search.exclude": {
-      "**/package-lock.json": true,
-      "**/locale": true
+      "**/package-lock.json": true
```

Comment:
package-lock.json files shouldn't longer exists on the monorepo as we're using pnpm

Reply:
right, I will ignore the pnpm lock instead